src/renderHyPhyVision.js

```diff
@@ -0,0 +1,41 @@
+import { default as render_absrel } from "./jsx/absrel.jsx";
+import { default as render_busted } from "./jsx/busted.jsx";
+import { default as render_fel } from "./jsx/fel.jsx";
+import { default as render_meme } from "./jsx/meme.jsx";
+import { default as render_prime } from "./jsx/prime.jsx";
+import { default as render_relax } from "./jsx/relax.jsx";
+import { default as render_slac } from "./jsx/slac.jsx";
+import { default as render_fubar } from "./jsx/fubar.jsx";
+import { default as render_gard } from "./jsx/gard.jsx";
+import { default as render_bgm } from "./jsx/bgm.jsx";
+import { default as render_fade } from "./jsx/fade.jsx";
+import { default as determineHyPhyMethod } from "./helpers/determineHyPhyMethod.js";
+import { default as render_inputErrorPage } from "./jsx/components/inputErrorPage.jsx";
+
+const methodDict = {
+  absrel: render_absrel,
+  busted: render_busted,
+  fel: render_fel,
+  meme: render_meme,
+  prime: render_prime,
+  relax: render_relax,
+  slac: render_slac,
+  fubar: render_fubar,
+  gard: render_gard,
+  bgm: render_bgm,
+  fade: render_fade
+};
+
+function renderHyPhyVision(data, element, fasta, originalFile, analysisLog) {
```

Comment:
For use in Galaxy, this needs to accept a URL as in here: https://github.com/veg/hyphy-vision/blob/master/src/jsx/template.jsx#L94

Reply:
You can cherry pick [this commit](https://github.com/stephenshank/hyphy-vision/commit/fa391e24da39977654b1d987ffa330434f7b6b63), which worked in Galaxy. Also, this should be a 2.4.0 release (I am currently linking in Galaxy with yalc).